tools/getkeys/index.js

```diff
@@ -131,34 +130,37 @@ class AzCliKeyVaultClient {
     }
 
     async getSecrets(vaultName) {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
         return JSON.parse(await execAsync(`az keyvault secret list --vault-name ${vaultName}`));
     }
 
     async getSecret(vaultName, secretName) {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
         return JSON.parse(await execAsync(`az keyvault secret show --vault-name ${vaultName} --name ${secretName}`));
     }
-};
+}
 
 class MsRestAzureKeyVaultClinet {
     static async get() {
-        const credentials = await msRestAzure.interactiveLogin();
+        const credentials = await interactiveLogin();
         return new MsRestAzureKeyVaultClinet(credentials);
     }
 
     constructor(credentials) {
-        this.client = new keyVault.KeyVaultClient(credentials);
+        this.client = new KeyVaultClient(credentials);
     }
 
     async getSecrets(vaultName) {
         return this.client.getSecrets(`https://${vaultName}.vault.azure.net/`);
     }
 
     async getSecret(vaultName, secretName) {
-        return this.client.getSecret(`https://${vaultName}.vault.azure.net/`, secretName, '');
+        return this.client.getSecret(`https://${vaultName}.vault.azure.net/`, secretName, "");
     }
 }
 
 async function getClient() {
+    // eslint-disable-next-line no-return-await
     return await AzCliKeyVaultClient.get();
```

Comment:
I think we can actually remove the comment and just make this `return AzCliKeyVaultClient.get()`. The suggestion is right, no need to await here, we can just return the promise since the caller will have to `await` the call to `getClient()` anyway.

Reply:
I see! Then removing all getClient() await here.